package.json

```diff
@@ -11,6 +11,7 @@
   "scripts": {
     "test": "NODE_ENV=MONGOTEST jest",
     "babel": "./node_modules/.bin/webpack -d --watch",
+    "build": "./node_modules/.bin/webpack -p",
```

Comment:
'babel' and 'build' are what i'd perceive as being confusing names for these scripts, considering one runs webpack in development and the other runs webpack in production.